### PR TITLE
CoreDNS 1.11.2 release notes and version bump

### PR DESCRIPTION
**Pull request by chrisohaver**
Initial 1.11.2 release notes and version bump

### CODE REVIEW

File: notes/coredns-1.11.2.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+description = "CoreDNS-1.11.2 Release Notes."
+tags = ["Release", "1.11.2", "Notes"]
+release = "1.11.2"
+date = "2024-01-26T00:00:00+00:00"
+author = "coredns"
++++
+
+This release contains some new features, bug fixes, and package updates.
+New features include:
+* When the _forward_ plugin receives a malformed upstream response that is overflowed,
```
Comment (bradbeam) [inline notes/coredns-1.11.2.md]: minor wording nit, 
```suggestion
* When the _forward_ plugin receives a malformed upstream response that overflows,
```

**Review by bradbeam — APPROVED** [review]

**Review by SuperQ — APPROVED** [review]

**Comment by zmedico** [comment]
The tag existed (01bded8194be73ce4601ad608a0464166cee932a) and then somehow went missing:

https://github.com/coredns/coredns/releases/tag/v1.11.2

**Comment by chrisohaver** [comment]
Tag will be recreated when it’s released. It’s not released yet.

**Comment by rahilarious** [comment]
> Tag will be created when it’s released. It’s not released yet.

There is a clear mishap from Coredns team. 1.11.2 was 100% released. Then you guys pulled it out. Many distros even updated to 1.11.2 https://repology.org/project/coredns/versions.

Because I'm "watching" this repo, I have the notification email I received upon 1.11.2 release on 26th Feb tagged at https://github.com/coredns/coredns/commit/8868454177bdd3e70e71bd52d3c0e38bcf0d77fd. Mistakes happen it's okay. But instead of pretending like nothing happened, own upto it

**Comment by chrisohaver** [comment]
> But instead of pretending like nothing happened, own upto it

@rahilarious,  its all **documented publicly in detail in the open 1.11.2 release issue**, which is still open and active. https://github.com/coredns/coredns/issues/6454

My terse answer was not a nefarious attempt to "pretend like nothing happened."

I removed the tag at the request of a community member. https://github.com/coredns/coredns/issues/6454#issuecomment-1996599800

**Pull request merged**